composer.json

```
@@ -28,7 +28,7 @@
     },
     "require": {
         "php": "^7.3 || ~8.0.0",
-        "dflydev/fig-cookies": "^2.0.1",
+        "dflydev/fig-cookies": "^2.0.1 || ~3.0.0",
```

I think it can uses : 

```suggestion
        "dflydev/fig-cookies": "^2.0.1 || ^3.0.0",
```